BeerProject/src/main/resources/db/version/003-AddUserTable.xml

```diff
@@ -9,15 +9,28 @@
             <column name="id" type="bigint" autoIncrement="true">
                 <constraints primaryKey="true"/>
             </column>
-            <column name="username" type="varchar(255)"/>
-            <column name="email" type="varchar(255)">
+            <column name="created_at" type="datetime"/>
+            <column name="update_date_time" type="datetime"/>
+            <column name="username" type="varchar(255)">
                 <constraints unique="true"/>
             </column>
             <column name="password" type="varchar(255)"/>
-            <column name="activated" type="boolean"/>
-            <column name="locked" type="boolean"/>
-            <column name="create_at" type="datetime"/>
-            <column name="updated_at" type="datetime"/>
+            <column name="email" type="varchar(255)"/>
+            <column name="is_account_non_expired" type="boolean"/>
+            <column name="is_account_non_locked" type="boolean"/>
+            <column name="is_credentials_non_expired" type="boolean"/>
+            <column name="is_enabled" type="boolean"/>
         </createTable>
+
+        <createTable tableName="user_granted_authorities">
+            <column name="user_id" type="bigint"/>
+            <column name="granted_authorities" type="varchar(255)"/>
+        </createTable>
+
+        <addForeignKeyConstraint baseColumnNames="user_id"
+                                 baseTableName="user_granted_authorities"
+                                 constraintName="FK5cxy225g3c11e2c3y1c45qlso"
```

Comment:
W jaki sposób generujesz "FK5cxy225g3c11e2c3y1c45qlso"? Tym UUID?

Reply:
pamietam że liquibase sam mi wygeneraował ten cia znaków a ja go później dopisałem. To raczej nie  jest poprawny tok działania, ale  wtedy taka opcja zadziałała i nie blokowała startu apki

Reply:
ok, rozumiem. Dzięki że powiedziałeś :)